### PR TITLE
wpc docs and minor improvements

### PR DESCRIPTION
**Pull request by mschroederi**

### CODE REVIEW

File: shift_detector/checks/word_prediction_check.py [hunk 1]
```diff
@@ -46,23 +46,33 @@ def run(self, store) -> Report:
...
+
+        for col, err in failure_columns.items():
+            explanation[col] = err
 
         return Report("WordPredictionCheck", examined_columns, shifted_columns, explanation)
 
     def detect_shifts(self, examined_columns: List[str], result: dict):
         shifted_columns = []
         explanation = {}
+        print('detect_shifts')
```
Comment (phSch08) [inline shift_detector/checks/word_prediction_check.py]: this could be something for our new custom print methods (e.g. lprint)

Comment (phSch08) [reply shift_detector/checks/word_prediction_check.py]: I don't think, that this statement has any value to the user

Comment (mschroederi) [reply shift_detector/checks/word_prediction_check.py]: just forgot to remove it

File: docs/source/checks/word_prediction.rst [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+Interpretation
+++++++++++++++
+
+The above report can be read as follows:
+
+1. The examined columns are 'Name', 'Entry', 'Type 1' and 'Type 2'.
+2. The column that contains a shift according to the word prediction check is 'Entry'.
+3. The loss increased from ~0.116 on test data from the first dataset to ~0.12 on the second dataset.
+   This is a increase of the loss of ~3.8%, which does not exceed the check's threshold.
```
[the remark this answers is not in the document]
Comment (mschroederi) [reply docs/source/checks/word_prediction.rst]: good point, I'll add that to the output

**Pull request merged**